spec/controllers/index_spec.rb

```diff
@@ -0,0 +1,78 @@
+require_relative '../spec_helper'
+
+describe 'Index Controller' do
+let (:new_sealion) {Sealion.create}
```

Comment:
Make sure you pass parameters into your create, just like you would need to in the controller.